contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -176,13 +180,17 @@ def __add__(self, other: 'Millisatoshi') -> 'Millisatoshi':
     def __sub__(self, other: 'Millisatoshi') -> 'Millisatoshi':
         return Millisatoshi(int(self) - int(other))
 
-    def __mul__(self, other: int) -> 'Millisatoshi':
+    def __mul__(self, other: Union[int, float]) -> 'Millisatoshi':
```

Comment:
Should probably raise an exception if we try to multiply msat with msat. Typing checks are not performed at runtime and devs often don't use `mypy` yet.